include/ndcurves/polynomial.h

```diff
@@ -282,6 +282,46 @@ struct polynomial : public curve_abc<Time, Numeric, Safe, Point> {
     return polynomial_t(coeffs, t_min, t_max);
   }
 
+  /**
+   * @brief MinimumJerk Build a polynomial curve connecting p_init to p_final
+   * minimizing the time integral of the squared jerk with a zero initial and
+   * final velocity and acceleration
+   * @param p_init the initial point
+   * @param p_final the final point
+   * @param t_min initial time
+   * @param t_max final time
+   * @return the polynomial curve
+   */
+  static void MinimumJerk(polynomial_t& out, const point_t& p_init,

```

Comment:
This should re-use the code of https://github.com/loco-3d/ndcurves/blob/5e980376bebeb103d006c70ae5c06bcbb95034bf/include/ndcurves/polynomial.h#L260 instead of duplicating the equations. Or the other way around, the existing method should create a curve and call this new method with it. 

Also, the docstring should specify that the curve need to be initialized with the correct size/degree.